readthedocs-theme/templates/includes/footer.html

```diff
@@ -0,0 +1,170 @@
+<footer class="ui basic inverted segment very padded" id="footer">
+  <div class="ui container">
+
+      <div class="ui three column stackable doubling grid information very padded">
+
+        <div class="column">
+          <h4 class="ui sub header">Stay Updated</h4>
+
+          <div class="ui list">
+            <a href="https://blog.readthedocs.com/" class="item">Blog</a>
+            <div class="item">
+              <a href="https://github.com/rtfd/readthedocs.org" rel="noopener noreferrer"><i class="icon large light github"></i></a>
+              <a href="https://twitter.com/readthedocs" rel="noopener noreferrer"><i class="icon large light twitter"></i></a>
+            </div>
+          </div>
+
+          <div class="ui form">
+            <form id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+              <p>Sign up for our newsletter to get our latest blog updates delivered to your inbox weekly.</p>
+
+              <div class="field">
+                <label for="mce-EMAIL">Email</label>
+                <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" placeholder="you@example.com">
+              </div>
+
+              <div id="mce-responses">
+                <div class="ui hidden response" id="mce-error-response"></div>
+                <div class="ui hidden response" id="mce-success-response"></div>
+              </div>
+
+              <input class="ui primary button" type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button">
+            </form>
+          </div>
+        </div>
+
+        <div class="column">
+          <h4 class="ui sub header">Get Involved</h4>
+
+          <div class="ui list">
+            <a class="item" href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html">Getting Started Guide</a>
+            <a class="item" href="https://docs.readthedocs.io">Documentation</a>
+            <a class="item" href="https://docs.readthedocs.io/page/contribute.html">Contributing</a>
+            <a class="item" href="https://docs.readthedocs.io/page/team.html">Team</a>
+            <a class="item" href="https://docs.readthedocs.io/page/open-source-philosophy.html">Open Source Philosophy</a>
+          </div>
+        </div>
+
+        <div class="column">
+          <h4 class="ui sub header">Business Info</h4>
+
+          <div class="ui list">
+            <a class="item" href="">Advertise with Us</a>
+            <a class="item" href="https://readthedocs.com">Read the Docs for Business</a>
+            <a class="item" href="">Supporters</a>
+            <a class="item" href="https://docs.readthedocs.io/page/privacy-policy.html">Privacy Policy</a>
+            <a class="item" href="https://docs.readthedocs.io/page/terms-of-service.html">Terms of Service</a>
+            <a class="item" href="https://read-the-docs-guidelines.readthedocs-hosted.com/">Branding</a>
```

Comment:
We should probably link to a documentation page rather than the repository. I'd be fine skipping this link for now until we have something more presentable than just a Git repo.

Reply:
This is the same on readthedocs.org now, so nothing lost. We can return to this too.